taxi/tests.py

```diff
@@ -2,2 +1,139 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
 
-# Create your tests here.
+from taxi.forms import CarForm, DriverLicenseUpdateForm
+from taxi.models import Manufacturer, Driver, Car
+
+
+class ModelTests(TestCase):
+    def setUp(self):
+        self.manufacturer = Manufacturer.objects.create(
+            name="manufacturer",
+            country="country"
+        )
+        self.driver = get_user_model().objects.create(
```

Comment:
Ensure that `get_user_model()` returns the `Driver` model or a compatible model. If `Driver` is not the user model, consider using `Driver.objects.create()` instead.